docs/content/v2024.2/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -226,3 +226,128 @@ Note that this requires all the zone deployments to be in the same namespace.
    ```
 
 After the service YAML is applied, in this example you would access the universe at `yb-k8s-common-tserver.yb-prod-yb-k8s.svc.cluster.local`.
+
+### Create a common load balancer service for YB-Masters/YB-TServers
+
+In versions v2.17 and later, newly created multi-zone universes are deployed in a single namespace by default. This can lead to duplication of load balancer services as a separate load balancer is created for each zone. In order to prevent creating extra load balancers, YBA provides the option to create a common load balancer service (currently {{<tags/feature/ea>}}) for YB-Masters and YB-TServers that spans across all the zones in a namespace.
```

Comment:
```suggestion
In v2.17 and later, newly created multi-zone universes are deployed in a single namespace by default. This can lead to duplication of load balancer services as a separate load balancer is created for each zone. To prevent creating extra load balancers, you can create a common load balancer service (currently {{<tags/feature/ea>}}) for YB-Masters and YB-TServers that spans all the zones in a namespace.
```